artifacts/install/apiserver-template.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: namespace-reservation-server
+parameters:
+- name: IMAGE
+  value: namespace-reservation-server:latest
+- name: NAMESPACE
+  value: openshift-namespace-reservation
+- name: LOGLEVEL
+  value: "0"
+objects:
+
+# to create the tsb server
+- apiVersion: extensions/v1beta1
+  kind: DaemonSet
+  metadata:
+    namespace: ${NAMESPACE}
+    name: server
+    labels:
+      server: "true"
+  spec:
+    template:
+      metadata:
+        name: server
+        labels:
+          server: "true"
+      spec:
+        serviceAccountName: server
```

Comment:
Request more specific than server throughout this template. We typically have to deploy all our apps into one namespace for a bunch of permissions/auditing/monitoring reasons in Online, would be helpful to more clearly see what belongs to this and what doesn't.

Reply:
> Request more specific than server throughout this template. We typically have to deploy all our apps into one namespace for a bunch of permissions/auditing/monitoring reasons in Online, would be helpful to more clearly see what belongs to this and what doesn't.

I strongly recommend against this.  It makes cleanup more difficult and effectively removes the benefit of separately ACL'd service accounts.  In addition, a transition to an ansible based installation would be more difficult.

Reply:
Ok will raise this with the team, we have apps that need access to metrics API and every namespace we use requires exclusion from a bunch of ops tooling as well as our own to make sure we don't inadvertently idle or archive something of ours. We should probably be doing that with a label on each project. Will add a trello card.

Reply:
I don't know how long that'll take or even how the discussion will go between team and ops, so when we go to deploy we may still need more specific naming for this thing.

Reply:
> Ok will raise this with the team, we have apps that need access to metrics API and every namespace we use requires exclusion from a bunch of ops tooling as well as our own to make sure we don't inadvertently idle or archive something of ours. We should probably be doing that with a label on each project. Will add a trello card.

This sounds like something that labeling would be a good fit for.